airgun/widgets.py

```diff
@@ -279,6 +279,27 @@ def fill(self, values):
                 self.assigned.fill(value)
         return True
 
+    def set_assigned(self, desired_values):
```

Comment:
not sure why that situation should happen. Are tests not isolated or you do things in one organization?

Reply:
This is a helper method so that I can pass in 1 list of values I want in the 'assigned' box and it takes care of un-assigning anything else that is in there

Reply:
Yeah, such situation can be required in some unique situation, when we had (at least I hope that we don't have them anymore) entities that were not organization specific or we cannot filter them in some way and you cannot predict what entities will be in the lists. But then, we still can assert that some entities are `in` not `equal`
Please, remove that method and we will add it in the future in case we face something described above